content/applications/productivity/discuss/email_servers.rst

```diff
@@ -84,6 +85,48 @@ that is a CNAME with the value "odoo._domainkey.odoo.com".
 
 .. _discuss/email_servers/inbound_messages:
 
+Choose allowed "From" email addresses
+-------------------------------------
+
+Sometimes, an email's "From" (outgoing) address can belong to a different
+domain, and that can be a problem.
+
+For example, if a customer with address *mary@customer.example.com* responds to
+a message, Odoo will try to redistribute that same email to other subscribers
+in the thread. But if the domain *customer.example.com* forbids that kind of
+usage for security (kudos for that), the Odoo's redistributed email would get
+rejected by some recipients' mail servers.
+
+To avoid those kind of problems, you should make sure all emails use a "From"
+address from your authorized domain.
+
+If your MTA supports `SRS (Sender Rewriting Scheme)
+<https://en.wikipedia.org/wiki/Sender_Rewriting_Scheme>`__, you can enable it
+to handle these situations. However, that is more complex and requires more
+technical knowledge that is not meant to be covered by this documentation.
+
+Instead, you can also configure Odoo to do something similar by itself:
+
+#.  Set your domain name in the General Settings.
+
+    .. image:: media/alias_domain.png
+      :align: center
```

Comment:
```suggestion
       :align: center
```
Nitpicking: https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#be-consistent-with-indentation